### DISCUSSION TITLE
how to access page metadata client side

### DISCUSSION THREAD

**Discussion by Jonathan-Peck** [facebook/docusaurus#7637] [Q&A]
we swizzled the DocBreadcrumbs component with the intention of adding a button that will redirect the browser somewhere. We would like to include as part of that redirect the current pages title. We tried accessing the props that are fed to the component, but it appears to be a null/blank object. 

what is the correct way to access the current pages metadata including page title?

**Comment by slorber** [comment]
Afaik we don't have a good way to do that currently. I'd like to put the current doc data into React context but until then you'll have to swizzle DocItem to pass extra props to the breadcrumb 😅

**Reply by slorber** [reply]
In this PR I add the doc data into React content: https://github.com/facebook/docusaurus/pull/7644

After merge you should be able to use the `useDoc()` hook to access any doc data you want in the breadcrumb
